packages/phone-number-privacy/src/index.ts

```diff
@@ -1,21 +1,148 @@
+import { AttestationsWrapper } from '@celo/contractkit/lib/wrappers/Attestations'
 import { BLINDBLS } from 'bls12377js-blind'
 import * as functions from 'firebase-functions'
+import fetch from 'node-fetch'
 import config, { connectToDatabase } from './config'
+import { ACCOUNTS_COLUMNS, ACCOUNTS_TABLE } from './models/account'
+import { getContractKit } from './web3/contracts'
+
+export const NUM_ATTESTATIONS_REQUIRED = 3
```

Comment:
Should this be configurable?

Reply:
its not in the mobile application, and I don't expect it to change much.